src/tracker/tracker-game-source.vala

```diff
@@ -0,0 +1,42 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private class Games.TrackerGameSource : Object, GameSource {
+	private Tracker.Sparql.Connection connection { private set; get; }
+	private GenericSet<TrackerQuery> queries;
+
+	public TrackerGameSource (Tracker.Sparql.Connection connection) {
+		this.connection = connection;
+	}
+
+	construct {
+		queries = new GenericSet<TrackerQuery> (direct_hash, direct_equal);
```

Comment:
I don't know, `direct_hash` and `direct_equal` are pretty standard IMHO, prefixing them with `GLib.` would be like prefixing `cout` with `std::` in C++.

I'm not against it except that if we want to do it here maybe we would want to do it for every GLib element, and I'm not sure it would improve readability.

But if you think we can make an exception here and prefix just these ones with `GLib.`, go on! =)